src/Compiler/CompilerOptions.cpp

```diff
@@ -309,6 +309,22 @@ llvm::cl::opt<std::string> modelTag("tag",
     llvm::cl::value_desc("a string that matches regex ([0-9a-z_.-]+)"),
     llvm::cl::init(""), llvm::cl::cat(OnnxMlirOptions));
 
+llvm::cl::opt<bool> enableConvOptPass("enable-conv-opt-pass",
+    llvm::cl::desc("Enable the ConvOptPass. Default is true."),
+    llvm::cl::init(true), llvm::cl::cat(OnnxMlirOptions));
+
+llvm::cl::list<std::string> extraLibDirs("extra-lib-dirs",
```

Comment:
Love this addition, was much needed. And thanks for making it a list, that is precisely what we need.

Can we call it "-L" and "-l"? As a group, we have added a lot of options. If there is a need to distinguish, then that is fine. Standard names require no new learning from normal users.

Reply:
Changed to use "-L" and "-l"